### PR TITLE
chore: fix formatting and spelling errors

### PR DESCRIPTION
**Pull request by taka-oyama**

### CODE REVIEW

File: tests/Query/DatetimeTest.php [hunk 1]
```diff
@@ -55,19 +58,19 @@ public function testTimezoneWithQueryBuilder(): void
         $conn = $this->getDefaultConnection();
 
         date_default_timezone_set('Asia/Tokyo');
-        $expected = new \DateTime('2018-03-13 09:00:00');
+        $expected = new DateTime('2018-03-13 09:00:00');
         $this->assertEquals('Asia/Tokyo', $expected->getTimezone()->getName());
 
         $row = $conn->query()->selectRaw('TIMESTAMP("2018-03-13T00:00:00Z")')->get()->first();
-        /** @var \DateTimeInterface $datetime */
+        /** @var DateTimeInterface $datetime */
```
Comment (tyabu12) [inline tests/Query/DatetimeTest.php:65]: I think `$this->assertInstanceOf(DateTimeInterface::class, $datetime)` is better than PHPDoc overwrite in view of reliability.

Comment (taka-oyama) [reply tests/Query/DatetimeTest.php:65]: That change goes against the title so it will be done in a different PR.

**Review by tyabu12 — APPROVED** [review]
LGTM

File: tests/Query/DatetimeTest.php (hunk 1 above)
Comment (taka-oyama) [reply tests/Query/DatetimeTest.php:65]: I think this makes sense but PHPDoc cannot be deleted because phpstan will throw an error here without it.

**Pull request merged**